tests/lighting_processing/test_lighting_processing.py

```diff
@@ -0,0 +1,445 @@
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*********************************************************************************************************
+"""
+import unittest
+from pathlib import Path
+
+from lxml import etree
+
+from buildingsync_asset_extractor.lighting_processing.lighting_processing import (
+    LightingDataLPD,
+    LightingDataPower,
+    process_buildings_lighting_systems
+)
+from buildingsync_asset_extractor.processor import BSyncProcessor
+
+SECTION_PATH = '/BuildingSync/Facilities/Facility/Sites/Site/Buildings/Building/Sections/Section'
+
+linked_section_with_floor_area_percentage = etree.XML('''
+    <acc:LinkedSectionID
+        IDref="Section-69928578013460"
+        xmlns:acc="http://buildingsync.net/schemas/bedes-auc/2019"
+    >
+        <acc:FloorAreas>
+            <acc:FloorArea>
+                <acc:FloorAreaType>Common</acc:FloorAreaType>
+                <acc:FloorAreaPercentage>50.0</acc:FloorAreaPercentage>
+            </acc:FloorArea>
+        </acc:FloorAreas>
+    </acc:LinkedSectionID>
+''')
+
+linked_section_with_floor_area_value = etree.XML('''
+    <acc:LinkedSectionID
+        IDref="Section-101919600"
+        xmlns:acc="http://buildingsync.net/schemas/bedes-auc/2019"
+    >
+        <acc:FloorAreas>
+            <acc:FloorArea>
+                <acc:FloorAreaType>Common</acc:FloorAreaType>
+                <acc:FloorAreaValue>50.0</acc:FloorAreaValue>
+            </acc:FloorArea>
+        </acc:FloorAreas>
+    </acc:LinkedSectionID>
+''')
+
+
+class TestBSyncProcessor(unittest.TestCase):
+    def setUp(self) -> None:
+        self.testfile = Path(__file__).parents[1] / 'files' / 'completetest.xml'
+        self.no_ns_testfile = Path(__file__).parents[1] / 'files' / 'testfile2.xml'
+        self.output_dir = Path(__file__).parents[1] / 'output'
+        self.out_file = 'testoutput.json'
+        self.out_file_2 = 'testoutput_2.json'
+        self.test_assets_file = Path(__file__).parents[1] / 'files' / 'test_asset_defs.json'
+        self.num_assets_to_extract = 23
+        self.num_sections_in_testfile = 3
+
+        # create output dir
+        self.output_dir.mkdir(parents=True, exist_ok=True)
+
+        print("TESTFILE: {}".format(self.testfile))
+        self.bp = BSyncProcessor(self.testfile)
+
+    def test_process_lighting_method_1(self) -> None:
+        # Set Up #
+        self.bp.process_sections()
+
+        # get lighting_systems and clear it
+        lighting_systems_path = "/BuildingSync/Facilities/Facility/Systems/LightingSystems"
+        lighting_systems = self.bp.xp(self.bp.doc, lighting_systems_path)[0]
+        for e in lighting_systems:
+            lighting_systems.remove(e)
+
+        # add method 1 lighting system
+        method_1_ls = etree.XML('''
+            <acc:LightingSystem xmlns:acc="http://buildingsync.net/schemas/bedes-auc/2019">
+                <acc:InstalledPower> 1.0 </acc:InstalledPower>
```

Comment:
spacing around the number looks strange--can you remove the extra spaces, unless there is a clear reason not to.